.github/workflows/php_lib_pull_requests.yml

```
@@ -63,13 +68,21 @@ jobs:
         with:
           php-version: ${{ matrix.php-version }}
       # Vendor PHP code.
-      - name: Composer
+      - name: Composer With Options
+        if: ${{ inputs.USE_COMPOSER_OPTIONS }}
         env:
           COMPOSER_AUTH: '${{ secrets.COMPOSER_AUTH }}'
         uses: "ramsey/composer-install@v2"
         with:
           working-directory: "${{ inputs.COMPOSER_WORKING_DIRECTORY }}"
           composer-options: "--ignore-platform-reqs"
```

Can this not be simplified to be:

```suggestion
          composer-options: "--ignore-platform-reqs=${{ inputs.USE_COMPOSER_OPTIONS }}"
```

---

looks like the --ignore-platform-reqs doesn't accept a value but I had used the composer-options as a string variable instead to make it cleaner